src/client/src/components/Graph.tsx

```diff
@@ -37,23 +38,33 @@ const graphStyle = {
         'linear-gradient(to bottom right, #00164f, #4e009c, #290066)',
 };
 
+interface TestInterface {
+    elements?: Elements;
+    selectedProject?: IProject;
+}
+
 export interface GraphProps {
-    setElements: React.Dispatch<React.SetStateAction<Elements>>;
+    test?: TestInterface;
```

Comment:
Naming something `test` is usually a bad practice, since it looks like you would be using that to pass in data in tests, while using some other way to pass in data in actual use, and you're not doing that, right? :)

It would be actually nice to refactor the Graph component into a pure component, which doesn't touch Redux and communicates through callbacks. It would make testing a lot nicer... but let's also do that later.